packages/contracts/build/contracts/IdentityEvents.json

```diff
@@ -876,7 +876,11 @@
     "name": "solc",
     "version": "0.4.25+commit.59dbf8f1.Emscripten.clang"
   },
-  "networks": {},
+  "networks": {
```

Comment:
Can we remove the contract JSON changes from this PR

Reply:
Agreed, removed